### PR TITLE
Adding escalation policy to the service

### PR DESCRIPTION
**Pull request by tkensiski**

### CODE REVIEW

**Comment by gmjosack** [comment]
I don't see that this has been added in the documentation. Do you have a reference for this being available?

File: pygerduty/__init__.py [hunk 1]
```diff
@@ -460,6 +460,7 @@ class Service(Container):
     def __init__(self, *args, **kwargs):
         Container.__init__(self, *args, **kwargs)
         self.email_filters = EmailFilters(self.pagerduty, self)
+        self.escalation_policy = EscalationPolicies(self.pagerduty, self)
```
Comment (gmjosack) [inline pygerduty/__init__.py]: Can you make this `self.escalation_policies` to have consistent pluralization with other Container instances.

**Comment by tkensiski** [comment]
its here: https://developer.pagerduty.com/documentation/rest/services/list

there is escalation policy -> object

File: pygerduty/__init__.py (hunk 1 above)
Comment (tkensiski) [inline pygerduty/__init__.py]: You can only have 1 escalation policy for a service so being plural would in my opinion be wrong

**Comment by gmjosack** [comment]
To get the escalation policy along with a service you would use the `include` parameter documented on the same page you listed.

```python
from pygerduty import PagerDuty
pd = PagerDuty("foocorp", "ABCDEFGHIJKLMNOP1234")

# Include on list queries
pd.services.list(
    query="My Service",
    include=["escalation_policy"]
).next()

# Include on show queries
pd.services.show(
    "ABCDEFG", include=["escalation_policy"]
)   
```

Nesting a `Collection` under a `Container` is meant for giving you access `show`/`list`/etc for a given returned resource. In this case it would try to hit a `services/:service_id/escalation_policy[/:escalation_id]` url which I don't see documented.

**Comment by tkensiski** [comment]
Ah, missed that part. Closing this PR

**Pull request closed without merging**